scripts/solc-bump.js

```diff
@@ -0,0 +1,56 @@
+const fs = require('fs');
+const { execSync } = require('child_process');
+const readline = require('readline');
+
+const PACKAGES_DIR = `${__dirname}/../packages`;
+
+/**
+ * Load packages depending on `solc`
+ **/
+function loadSolcDependentPackages() {
+  const packages = fs.readdirSync(PACKAGES_DIR);
+  return packages.filter(packageName => {
+    // parse the package.json file for `packageName`
+    const packagePath = `${PACKAGES_DIR}/${packageName}/package.json`;
+    const packageJson = JSON.parse(fs.readFileSync(packagePath, 'utf8'));
+
+    // operate if the solc dependency exists
+    return packageJson
+      && packageJson.dependencies
+      && !!packageJson.dependencies['solc'];
```

Comment:
We might want to include `devDependencies` here; any thoughts?

Reply:
Agreed, updated and refactored a bit